src/backend/starknet.cairo

```diff
@@ -96,7 +96,7 @@ namespace Starknet {
 
     // @notice Populate a Environment with Starknet syscalls
     func get_env{syscall_ptr: felt*}(
-        origin: model.Address*, gas_price: felt
+        origin_evm_address: felt, gas_price: felt
```

Comment:
I don't really like the `_evm_address` suffix everywhere. I think that we should consider that addresses are EVM address unless explicitely said to be something else